Makefile

```diff
@@ -4,4 +4,8 @@ install:
 	poetry install
 
 test:
-	poetry run python -m pytest -vvv
+	poetry run pytest \
```

Comment:
I have no memory of why `nbmake` would be required as its for notebooks and we have none in this repo, I reckon we stick to the original make command and remove it as a dependency.